Providing a flag without any command causes a panic

### Summary

An unexpected panic happens if the `etime` command is used with only flag values and no command:

```sh
$ ./etime -p
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/zimeg/emporia-time/pkg/times.parseTimeResults({{0xc000452400, 0x0, 0x200}, 0x0, 0x0})
        ~/go/src/github.com/zimeg/emporia-time/pkg/times/time.go:87 +0x40f
github.com/zimeg/emporia-time/pkg/times.TimeExec({{0xc0000c2010, 0x0, 0x0}, {{0x0, 0x0}, {0x0, 0x0}, {0x0, 0x0}, 0x1, ...}})
        ~/go/src/github.com/zimeg/emporia-time/pkg/times/time.go:53 +0x3e5
main.main()
        ~/go/src/github.com/zimeg/emporia-time/main.go:42 +0x2d8
```

### Expected result

Gracefully errors and exits with a helpful message.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Parse the provided help flag to display a helpful message
 - Return the actual error that happens when building templates
+- Avoid a panic when no command arguments are provided
 
 ### Maintenance
 

--- a/internal/program/flags.go
+++ b/internal/program/flags.go
@@ -37,8 +37,9 @@ func ParseFlags(arguments []string) Command {
 
 	flagset.Usage = templates.PrintHelpMessage
 	flagset.Parse(arguments[1:])
-	if len(arguments) <= 1 {
+	commandArgs := flagset.Args()
+	if len(commandArgs) <= 0 {
 		flags.Help = true
 	}
-	return Command{Args: flagset.Args(), Flags: flags}
+	return Command{Args: commandArgs, Flags: flags}
 }

--- a/internal/program/flags_test.go
+++ b/internal/program/flags_test.go
@@ -64,6 +64,13 @@ func TestParseFlags(t *testing.T) {
 				Flags: Flags{Help: true},
 			},
 		},
+		"help is noticed when no commands are provided": {
+			arguments: []string{"etime", "-p"},
+			command: Command{
+				Args:  []string{},
+				Flags: Flags{Help: true, Portable: true},
+			},
+		},
 	}
 	for name, tt := range tests {
 		t.Run(name, func(t *testing.T) {

